bodhi/server/scripts/approve_testing.py

```diff
@@ -68,6 +67,8 @@ def main(argv=sys.argv):
     if len(argv) != 2:
         usage(argv)
 
+    logging.basicConfig(level=logging.ERROR)
```

Comment:
Hmm, if you do just that, what happens when the `main()` function is called in the unit tests? Does it change the log level globally for all the tests?

Reply:
Yes, it would, but my latest changes address this.